gcloud-java-logging/src/test/java/com/google/cloud/logging/LoggingLevelTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.logging;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import java.util.logging.Level;
+
+public class LoggingLevelTest {
+
+  @Test
+  public void testDebug() {
+    LoggingLevel debug = LoggingLevel.DEBUG;
+    assertEquals(Severity.DEBUG, debug.severity());
+    assertEquals("DEBUG", debug.getName());
+    assertTrue(debug.intValue() < Level.FINEST.intValue());
+    assertTrue(debug.intValue() > Level.ALL.intValue());
+  }
+
+  @Test
```

Comment:
I disagree. This class checks that the additional logging levels that we define in the `LoggingLevel` class are correctly initialized. Initialization here is very important as it defines how those levels relate to the original JUL levels (i.e. whether they are more or less severe). This will affect "loggability" at both logger and handler levels, as well as flushing in some cases. I believe it's rather important that we check that we initialize these extra levels properly. 

Also they JUL levels as well as our "extra" levels are not enums but instance of the `Level` class and need a name to be explicitly provided via constructor (hence this test checks that the expected string was provided).